src/bugjira/config.py

```diff
@@ -1,38 +1,35 @@
 import json
 
-from pydantic import BaseModel, validator, constr
+from pydantic import BaseModel, ConfigDict, field_validator, constr
 
 
 class BugzillaConfig(BaseModel):
+    model_config = ConfigDict(extra='forbid')
+
     URL: constr(strip_whitespace=True, min_length=1)
     api_key: constr(strip_whitespace=True, min_length=1)
 
-    class Config:
-        extra = 'forbid'
-
 
 class JiraConfig(BaseModel):
+    model_config = ConfigDict(extra='forbid')
+
     URL: constr(strip_whitespace=True, min_length=1)
     token_auth: constr(strip_whitespace=True, min_length=1)
 
-    class Config:
-        extra = 'forbid'
-
 
 class ConfigDict(BaseModel):
+    model_config = ConfigDict(extra='forbid')
```

Comment:
oops! good catch.

Reply:
I was wondering as well... and I think that's why we have to "import pydantic" instead of just having a specific class import on the next line, like for the other imports? After I asked the question, I realised it would conflict with our custom class, but would agree otherwise on maybe renaming for clarity in a later patch, if that is possible.